bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70-en.html

```diff
@@ -46,21 +51,23 @@ <h2 class="c-emphasis-box-title">
         <div class="show-fxa-supported-signed-out show-fxa-default show-fxa-not-fx">
           {{ monitor_button(
             entrypoint=_entrypoint,
+            entrypoint_experiment=_entrypoint_experiment,
```

Comment:
Both `_entrypoint_experiment` and `_entrypoint_variation` are undefined here. It doesn't seem to break anything, but should we set a default value of `=None` if the template isn't being extended for an experiment?

e.g. `{% set _entrypoint_experiment = None %}`